apps/frontend/src/libs/constants/sidebar-items.constant.ts

```diff
@@ -0,0 +1,17 @@
+import wheelOfBalance from "~/assets/img/wheel-of-balance-active.svg";
+import wheelOfBalanceInactive from "~/assets/img/wheel-of-balance-inactive.svg";
+import { AppRoute } from "~/libs/enums/enums.js";
+import { SidebarItem } from "~/libs/types/sidebar-item.type.js";
+
+const sidebarItems: SidebarItem[] = [
+	{
+		href: AppRoute.ROOT,
+		icon: {
+			active: wheelOfBalance,
```

Comment:
Let's create a separate `Icon` component, which will accept icon name
Here you will have icon name

Reply:
I'll use item.title for key then

Reply:
By here you mean active: IconName in type definition?

Reply:
type SidebarItem = {
	href: ValueOf<typeof AppRoute>;
	icon: {
		active: IconName;
		inactive: IconName;
	};
	title: string;
};

Reply:
<img width="325" alt="Screenshot 2024-08-23 at 11 59 54" src="https://github.com/user-attachments/assets/0195ac59-b378-4a0a-bb8c-f72e344e5a78">

Reply:
The reason why I have active and inactive in icon names is because those are intended to be two separate svg files under the hood

Reply:
Should this Icon component be passed as a child prop to SIdebarLink or used inside SidebarLink's JSX?

Reply:
The type you sent seems to be correct. To SidebarLink component we will send IconName, which will be used to show the icon